pulp_ci_centos/Containerfile

```diff
@@ -98,8 +104,12 @@ COPY assets/pulpcore-api.prep /etc/cont-init.d/pulpcore-api
 COPY assets/pulpcore-api.run /etc/services.d/pulpcore-api/run
 COPY assets/000-postgres.prep /etc/cont-init.d/000-postgres
 COPY assets/zzz-postgres.prep /etc/cont-init.d/zzz-postgres
-COPY assets/nginx.conf /etc/nginx/nginx.conf
+COPY assets/ssl_nginx.conf /nginx/ssl_nginx.conf
+COPY assets/nginx.conf /nginx/nginx.conf
 COPY assets/nginx.run /etc/services.d/nginx/run
+COPY assets/webserver.sh /nginx/webserver.sh
+
+RUN /nginx/webserver.sh
```

Comment:
This will use the $SCHEME buildarg.
Maybe make it a positional argument.

Also would it be possible to not perform all the ssl setup (especially the cert generation) in the http container?

Reply:
I'll be merging this PR and I'll try to address that on a follow-up PR